xbmc/guilib/GUIFontManager.h

```diff
@@ -40,6 +42,17 @@ struct OrigFontInfo
   bool border;
 };
 
+struct FontMetadata
+{
+  FontMetadata(const std::string& filename, const std::string& familyName)
+    : m_filename(filename), m_familyName(familyName)
```

Comment:
Out of curiosity, does default initialization with {} work on these cases? I mean `: m_filename{filename}, m_familyName{familyName}`? I suppose it doesn't

Reply:
yes should works, i have just read c++ guidelines

Reply:
If it works we should use it, I always found that format of initialization for members in the base class extremely confusing as it looks like those members are functions or something similar